src/Processors/Executors/AsyncTaskQueue.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include <cstddef>
+#include <cstdint>
+#include <mutex>
+#include <atomic>
+#include <unordered_map>
+
+namespace DB
+{
+
+#if defined(OS_LINUX)
+class AsyncTaskQueue
```

Comment:
Sounds too generic, is it a `RemoteIOQueue` actually? Or `EPollQueue` or something.

Reply:
Done